posthog/gitsha.py

```diff
@@ -0,0 +1 @@
+GIT_SHA = "Undefined"
```

Comment:
This is the wrong way to do this. We have settings.py for this + environment variables.

Reply:
Exactly as Michael calls out here as well it seemed risky to change a file that can be changed in various ways. Though I should have left a comment at the top of this file explaining that it gets overwritten by our git workflows to make things super clear.